### PR TITLE
add example biosnoop

### PR DESCRIPTION
**Pull request by ethercflow**

### CODE REVIEW

File: examples/biosnoop.c [hunk 1]
```diff
@@ -0,0 +1,107 @@
+#include <uapi/linux/ptrace.h>
```
Comment (brayniac) [inline examples/biosnoop.c]: If this code is taken from, or even just inspired by, code found in the BCC repo, we should cite it here. Example: https://github.com/rust-bpf/rust-bcc/blob/master/examples/softirqs.c#L3

File: examples/biosnoop.rs [hunk 2]
```diff
@@ -0,0 +1,136 @@
+use bcc::core::BPF;
```
Comment (brayniac) [inline examples/biosnoop.rs]: Same comment about citing example in the BCC repo if applicable. See https://github.com/rust-bpf/rust-bcc/blob/master/examples/softirqs.rs#L9

File: examples/biosnoop.c (hunk 1 above)
Comment (ethercflow) [reply examples/biosnoop.c]: Oh I am quite sorry about this, I'll fix this and pay attention later.

**Comment by ethercflow** [comment]
Done，PTAL.

**Comment by brayniac** [comment]
Thanks for the PR!

**Pull request merged**